tests/test_core.py

```diff
@@ -174,120 +174,98 @@ def test_stage_shutdown():
     assert len(test_all) == 2
 
 
-class TestHomeAssistant(unittest.TestCase):
-    """Test the Home Assistant core classes."""
+async def test_pending_sheduler(hass):
+    """Add a coro to pending tasks."""
+    call_count = []
 
-    # pylint: disable=invalid-name
-    def setUp(self):
-        """Set up things to be run when tests are started."""
-        self.hass = get_test_home_assistant()
+    async def test_coro():
+        """Test Coro."""
+        call_count.append("call")
 
-    # pylint: disable=invalid-name
-    def tearDown(self):
-        """Stop everything that was started."""
-        self.hass.stop()
+    for _ in range(3):
+        hass.async_add_job(test_coro())
 
-    def test_pending_sheduler(self):
-        """Add a coro to pending tasks."""
-        call_count = []
+    await asyncio.wait(hass._pending_tasks)
 
-        @asyncio.coroutine
-        def test_coro():
-            """Test Coro."""
-            call_count.append("call")
+    assert len(hass._pending_tasks) == 3
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6072450)